manim/mobject/types/vectorized_mobject.py

```diff
@@ -911,12 +911,15 @@ class VDict(VMobject):
     A VGroup-like class, also offering submobject access by
     key, like a python dict
     '''
-    def __init__(self, *pairs, **kwargs):
+
+    def __init__(self, *pairs, show_keys=False, **kwargs):
```

Comment:
Could you please add docstrings to this class and all its methods following [the format detailed in the wiki](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))?

Summoning Captain Docs : @PgBiel

Reply:
I've added the docstrings. Please let me know if any changes have to be made.